src/components/publishing/__test__/fixtures.tsx

```diff
@@ -93,11 +93,21 @@ export const ImageSetFull: ImageSetPreviewProps["section"] = {
 
 export const ImageSetMini: ImageSetPreviewProps["section"] = {
   type: "image_set",
-  layout: "mini",
   title: "The Work of Bruce M. Sherman",
   images: Images,
 }
 
+export const ImageSetFullSansTitle: ImageSetPreviewProps["section"] = {
+  type: "image_set",
+  layout: "full",
+  images: Images,
+}
+
+export const ImageSetMiniSansTitle: ImageSetPreviewProps["section"] = {
+  type: "image_set",
+  images: Images,
+}
```

Comment:
The pattern I've been using for supporting various fixtures is to create an `ImageSetPreview` array that contains the different types of image sets (ie: Image or HeroSection). This totally works fine too, but it might be confusing later on if we're using different patterns and maybe we should stick to one? What's the reason you separated it out for the ImageSet?